Makefile

```diff
@@ -3,9 +3,11 @@ start: api
 test:
 	make testunit && make testapi
 
-testunit: composer rmTestDb upTestDb yiimigratetestDb yiimigratetestDblocal rmTestDb
+testunit: composer emailcron rmTestDb upTestDb yiimigratetestDb yiimigratetestDblocal
+	docker-compose run emailcron whenavail emaildb 3306 100 ./yii migrate --interactive=0
```

Comment:
This line seems redundant, since the new `emailcron` entry in the line above will automatically run `run-cron.sh`, which runs the database migrations (albeit without the use of `whenavail`).